File: square/src/main/java/com/raxdenstudios/square/interceptor/autoinflatelayout/AutoInflateLayoutInterceptorCallback.java
package com.raxdenstudios.square.interceptor.autoinflatelayout;

import android.os.Bundle;
import android.view.View;

import com.raxdenstudios.square.interceptor.InterceptorCallback;

/**
 * Created by Ángel Gómez on 22/05/2015.
 */
public interface AutoInflateLayoutInterceptorCallback extends InterceptorCallback {

    void onContentViewCreated(View view, Bundle savedInstanceState);

}

File: square-commons/src/main/java/com/raxdenstudios/square/interceptor/commons/autoinflatelayout/AutoInflateLayoutInterceptorCallback.kt
package com.raxdenstudios.square.interceptor.commons.autoinflatelayout

import android.os.Bundle
import android.view.View

import com.raxdenstudios.square.interceptor.InterceptorCallback

/**
 * Created by Ángel Gómez on 22/05/2015.
 */
interface AutoInflateLayoutInterceptorCallback : InterceptorCallback {

    fun onContentViewCreated(view: View, savedInstanceState: Bundle?)

}
